_partials/cartBox/item_option_checkbox.php

```diff
@@ -20,7 +20,7 @@ class="custom-control-label w-100"
             for="menuOptionCheck<?= $optionIndex; ?>"
         >
             <?= $optionValue->name; ?>
-            <span class="pull-right"><?= currency_format($optionValue->price); ?></span>
+            <?php if ($optionValue->price > 0 || $hideFreeOptionPrices == false) {?><span class="pull-right"><?= currency_format($optionValue->price); ?></span><?php } ?>
```

Comment:
Also for readability, change the if statements to `$optionValue->price > 0 || !$hideZeroOptionPrices`. It's also missing the + sign.

Reply:
Done :)